packages/libs/eda/src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -198,6 +198,29 @@ export interface ScatterPlotDataWithCoverage extends CoverageStatistics {
 // define ScatterPlotDataResponse
 type ScatterPlotDataResponse = ScatterplotResponse;
 
+// define dataSetProcess type used inside the processInputData()
+// Note that dataSetProcess is different from the one used in the outside of the processInputData()
+// using pre-existing ScatterPlotDataSeries to override
+type Override<T1, T2> = Omit<T1, keyof T2> & T2;
+type dataSetProcessType = Override<
```

Comment:
Please start types with an uppercase letter
```suggestion
type DataSetProcessType = Override<
```

Reply:
Oops! I thought it was capital :( Changed it